src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,61 @@
 package core.basesyntax;
 
+// using abstract class
+
 public class MainApp {
 
+    abstract class Machine {
```

Comment:
let's make these inner classes separate classes

Reply:
I made this classes separately but maven need that this classes will be in separate files.

Reply:
> I made this classes separately but maven need that this classes will be in separate files.



> 

I made this classes separately but maven need that this classes will be in separate files.